compiler/luci/import/src/Importer.cpp

```diff
@@ -315,8 +315,14 @@ std::unique_ptr<Module> Importer::importModule(const circle::Model *model) const
 
   post_import_graph(module.get(), reader);
 
-  // Initialize 'source_table'
+  // Initialize 'memory_plan_table'
   auto circle_metadata = std::make_unique<luci::CircleImportMetadata>(reader);
+  if (circle_metadata->memory_plan_table().size() > 0)
+  {
+    module->memory_plan(circle_metadata->memory_plan_table());
+  }
+
```

Comment:
@binarman,  As I understand code from compiler/luci/lang/include/luci/IR/Module.h should be removed 
> I think code in this file is redundant, could you remove it?

that is why code from this file also should be removed, right? 


But how will we pass the information to the module?

Reply:
> But how will we pass the information to the module?

Just by reading this, (not know other things discuessed), we can introduce new header for this where both modules can access. maybe in new `luci/plan` folder for planning stuff.